### PR TITLE
feat: 🎸 erctl

### PR DESCRIPTION
**Pull request by haohanyuzmx**

### CODE REVIEW

File: pkg/agent/getterserver/rule.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	"github.com/everoute/everoute/pkg/agent/datapath"
+	"github.com/everoute/everoute/pkg/apis/rule/v1alpha1"
+	"github.com/everoute/everoute/pkg/constants"
+)
+
+type Getter struct {
+	dpManager *datapath.DpManager
+}
+
+func (g *Getter) GetAllRules(context.Context, *emptypb.Empty) (*v1alpha1.RuleEntries, error) {
```
Comment (KeyongSun) [inline pkg/agent/getterserver/rule.go:21]: Between each function add a empty line.

File: pkg/agent/getterserver/rule.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		if err != nil {
+			klog.Fatalf("remove remaining sock file error, err:%s", err)
+			return
+		}
+	}
+	// listen socket
+	listener, err := net.Listen("unix", constants.GetterRPCSocketAddr)
+	if err != nil {
+		klog.Fatalf("Failed to bind on %s: %v", constants.GetterRPCSocketAddr, err)
+	}
```
Comment (wu-cl) [inline pkg/agent/getterserver/rule.go:51]: Maybe we could reuse the original RPC server, instead of a new domain socket?

Comment (haohanyuzmx) [reply pkg/agent/getterserver/rule.go:51]: rpcserver的listener没有暴露出去，把getter服务的注册移动到rpcserver内吗

Comment (wu-cl) [reply pkg/agent/getterserver/rule.go:51]: Yes, `rpcserver/server.go` is intended to listen socket and register service.

**Review by KeyongSun — COMMENTED** [review]
format the code style, add space line between functions.

**Comment by haohanyuzmx** [comment]
> format the code style, add space line between functions.

刚刚改了，等下一次推

**Pull request closed without merging**